test/spec_session_abstract_session_hash.rb

```diff
@@ -25,4 +25,17 @@ def session_exists?(req)
     assert_equal [:bar, :qux], hash.values
   end
 
+  describe "#fetch" do
+    it "returns value for a matching key" do
```

Comment:
It is missing a test where you don't have a default block or value and the value is unknow